movingTech.NY/Driver/DevicePermission.java

```diff
@@ -68,8 +74,20 @@ private void drawOverApplicationsFlow() throws InterruptedException {
 			driver.findElement(AppiumBy.xpath("//android.widget.TextView[@content-desc='Search']")).click();
 			Thread.sleep(2000);
 			driver.findElement(AppiumBy.xpath("//android.widget.AutoCompleteTextView[@text='Search']")).sendKeys("Namma");
+			
+			List<WebElement> appName = driver.findElements(AppiumBy.xpath("//android.widget.TextView[@text='Namma Yatri Partner']"));
+
+			if(!appName.isEmpty() && appName.get(0).isDisplayed()) {
+				System.out.println("App name displyed");
+				appName.get(0).click();
+				}
+			else {
+				driver.findElement(AppiumBy.xpath("//android.widget.TextView[@text='in.juspay.nammayatripartner.debug']")).click();
+				System.out.println("App ackage name displyed");
```

Comment:
Can we check the spelling of package and displayed here?

Reply:
Sure, I will change